modules/items/client/controllers/admin/cats.client.controller.js

```diff
@@ -13,20 +13,10 @@
     vm.cat = cat;
     vm.authentication = Authentication;
     vm.form = {};
-    vm.remove = remove;
     vm.create = create;
     vm.list = list;
 
     // Remove existing Item
```

Comment:
if we are deleting the function, we also need to delete the comment

Reply:
Yep